src/pools/cluster/fixed.ts

```diff
@@ -58,7 +58,8 @@ export class FixedClusterPool<
     return isMaster
   }
 
-  protected destroyWorker (worker: Worker): void {
+  /** @inheritdoc */
+  public destroyWorker (worker: Worker): void {
```

Comment:
Is this public for a particular reason now ?
Did we tracked this change into the change log?

Reply:
Yes and no

We needed to open it (https://github.com/pioardi/poolifier/pull/191#issuecomment-782707329)
And no we havn't added this to the changelog yet

Reply:
Dynamic pool worker choice strategy require to access some methods in the abstract class. 
Not sure it will last if we make strategies code not manipulate the pool internals, but for now it's the less ugly solution without a complete rewrite of the dynamic worker choice code.